containerz/containerz.proto

```diff
@@ -118,6 +119,23 @@ service Containerz {
 
   // ListVolume returns all volumes that match the spec defined in the filter.
   rpc ListVolume(ListVolumeRequest) returns (stream ListVolumeResponse) {}
+
+  // StartPlugin starts the specified plugin identified by the name field
+  // of the request. It should be started using the instance_name of the
+  // request. If the plugin file named `name.tar` does not exist, this
+  // should return an error.
+  rpc StartPlugin(StartPluginRequest) returns (StartPluginResponse) {}
```

Comment:
Given we have separate plugin-specific commands for start/stop/list -- did you consider having a different one for `Deploy` to make the API consistent? (Not sure of the pros/cons.)

Is there a link to how volume plugins look for podman? Best I could find is https://github.com/containers/podman/discussions/23762.

Reply:
AFAIK Docker plugins can be used in podman so podman plugins are the same as docker plugins. 

I thought about a separate Deploy API but it seems like a lot of duplication since Deploy already uploads a file as we need.